health/warnings.go

```diff
@@ -18,17 +21,37 @@ var updateAvailableWarnable = Register(&Warnable{
 	Title:    "Update available",
 	Severity: SeverityLow,
 	Text: func(args Args) string {
-		return fmt.Sprintf("An update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		if version.IsMacAppStore() || version.IsAppleTV() {
+			return fmt.Sprintf("An update from version %s to %s is available. Open the App Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsMacSys() {
+			return fmt.Sprintf("An update from version %s to %s is available. Update now from the Tailscale settings.", args[ArgCurrentVersion], args[ArgAvailableVersion])
```

Comment:
"the"? I guess that's in the menu bar?

I'd just drop that second sentence and instead make the GUI recognize that update code and link it to the right thing?

Reply:
dropped